CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-20.5.1
+
+* CLI
+  * Fixed all commands to use kubeconfig's default namespace if specified
+    (thanks @Matei207!)
+  * Added multicluster checks to the `linkerd check` command
+  * Hid development flags in the `linkerd install` command for release builds
+* Controller
+  * Added ability to configure Prometheus Alertmanager (more documentation and
+    changes to come for this!) (thanks naseemkullah!)
```

Comment:
Missed the @ for the user handle

Reply:
Also, added ability to configure alerting and recording rules using external configmaps, and ability to add more flags to the prometheus command.

Reply:
@alpeb None of this is really documented or explained much in the PR. I didn't clarify more here because even if we mentioned it, I don't think we can really point users to the PR description. I can include it if you think it should be, but does that make sense?

Reply:
The PR addressed four points: alertmanager, alert rules, recording rules and command flags, but you only mentioned the first one. I think we should mention them all, without pointing them to the PR. There are some commented-out examples in `values.yaml` for those new configs, but full docs will follow along with the BYOP project.

Reply:
So to give more hints to users, you could mention those are new config values just for helm, and so people should know they can look into the chart's readme or values.yaml for more info.

Reply:
Okay I split this up into two separate bullets. Let me know your thoughts